libs/common/frontend/models/src/lib/services/authentication.service.ts

```diff
@@ -32,8 +32,6 @@ export class AuthenticationService {
 
 	login(email: string, password: string) {
 		if (this.checkCredentials(email, password)) {
-			const token = this.generateToken(email, password);
-			localStorage.setItem(this.tokenKey, token);
```

Comment:
removing these lines breaks the login. It is necessary for the token to be stored in the localstorage so you are logged in even after refreshing the page

Reply:
Oh damn sorry the login didnt work for me and therefore i disabled it. However forgot to revert the change.

Reply:
Ok np, reasons why it might not work:
- wrong credentials: email: your mail, password: siemens 
(maybe try my email: ingo.sternberg@fau.de)
- no secret key in .env file